controller/haproxy/api/frontend.go

```diff
@@ -51,7 +51,7 @@ func (c *clientNative) FrontendEdit(frontend models.Frontend) error {
 	return c.nativeAPI.Configuration.EditFrontend(frontend.Name, &frontend, c.activeTransaction, 0)
 }
 
-func (c *clientNative) FrontendEnableSSLOffload(frontendName string, certDir string, alpn bool) (err error) {
+func (c *clientNative) FrontendEnableSSLOffload(frontendName string, certDir string, alpn bool, alpnValue string) (err error) {
```

Comment:
I think there is no need to have both `alpn bool` and `alpnValue string`
We can have a single param `alpn string` where `if alpn != ""` then controller sets the alpn field.

Reply:
Done, thanks for the suggestion